runners/spark/3/src/main/java/org/apache/beam/runners/spark/structuredstreaming/translation/batch/DoFnMapPartitionsFactory.java

```diff
@@ -136,15 +211,15 @@
     }
 
     @Override
-    protected OutputT computeNext() {
+    protected OutT computeNext() {
```

Comment:
it was already clear but now if a user asks the difference between a Beam bundle and a spark partition, I could answer "none" and point to this code that invokes startBundle/finishBundle with each spark partition.